scripts/test/lint.py

```diff
@@ -86,6 +86,26 @@ def run_cmd(cmd):
     return out
 
 
+class IncludesDescriptionLength(Lint):
+    max_length = 175
+    min_length = 10
+    name = "the description length is invalid"
+    recommendation = "See the VM-Packages Wiki for conventions and package structure related to the description"
+
+    def check(self, path):
+        dom = minidom.parse(str(path))
+        metadata = dom.getElementsByTagName("metadata")[0]
+        description = metadata.getElementsByTagName("description")[0]
+        text = description.firstChild.data
+        length = len(text)
+        if not (self.min_length <= length <= self.max_length):
+            print(
+                f"Description has length {length}, which is outside the range [{self.min_length}, {self.max_length}]."
+            )
```

Comment:
[nitpick] I think we should remove the range as it should be displayed in the recommendation

```suggestion
            print(
                f"The description has invalid length: {len(description)}")
```